### PR TITLE
Review docs notes

### PR DESCRIPTION
**Pull request by HChughtai**
Added some clarifications including where to go to install Sphinx from, as well as updated a broken link to `sphinx-quickstart` documentation.

### CODE REVIEW

File: ch04packaging/04documentation.ipynb [hunk 1]
```diff
@@ -134,15 +134,22 @@
...
+    "Install Sphinx using the [appropiate instructions](https://www.sphinx-doc.org/en/master/usage/installation.html) for your system following the documentation online."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "\n",
-    "Invoke the [sphinx-quickstart](https://www.sphinx-doc.org/en/latest/usage/quickstart.html) command to build Sphinx's\n",
+    "Invoke the [sphinx-quickstart](https://www.sphinx-doc.org/en/master/usage/quickstart.html) command to build Sphinx's\n",
```
Comment (dpshelio) [inline ch04packaging/04documentation.ipynb:152]: Why have they got rid of `latest`? 😠  The problem if we refer to `master` is that we are pointing to the unreleased version of sphinx... and the documentation may not match with the version they have installed. Hopefully they won't change much the quickstart tool.

**Review by dpshelio — APPROVED** [review]
All looks very good to me! Thanks @HChughtai

File: ch04packaging/04documentation.ipynb (hunk 1 above)
Comment (HChughtai) [reply ch04packaging/04documentation.ipynb:152]: It is a little annoying. We could point it at the `3.x` version instead as that's likely the version they'll have if they installed it recently?

Comment (ageorgou) [reply ch04packaging/04documentation.ipynb:152]: We can add a comment that the output they see from the quickstart tool may vary a bit from what's in the notes or the instructions we point them to.

**Review by ageorgou — APPROVED** [review]
Great!

File: ch04packaging/04documentation.ipynb (hunk 1 above)
Comment (HChughtai) [reply ch04packaging/04documentation.ipynb:152]: Sounds good - I'll add that and then merge.

**Pull request merged**